argzoo/cifar10/cifar10_fedcac_resnet_config.py

```diff
@@ -0,0 +1,60 @@
+from easydict import EasyDict
+import argparse
+
+def args_parser():
```

Comment:
Remove these argument parser. Just use the hyperparameters recommended in the original paper.